--- a/lib_fp/Custom_Button_Switch_SMD.pretty/SW-SMD_L3.9-W3.0-P4.45.kicad_mod
+++ b/lib_fp/Custom_Button_Switch_SMD.pretty/SW-SMD_L3.9-W3.0-P4.45.kicad_mod
@@ -149,13 +149,13 @@
 	)
 	(pad "1" smd rect
 		(at -2.18 0)
-		(size 1.23 1.36)
+		(size 1.5 1.6)
 		(layers "F.Cu" "F.Paste" "F.Mask")
 		(uuid "f9bd8b39-ccf7-41a3-a1a1-50ed17c14d58")
 	)
 	(pad "2" smd rect
 		(at 2.18 0)
-		(size 1.23 1.36)
+		(size 1.5 1.6)
 		(layers "F.Cu" "F.Paste" "F.Mask")
 		(uuid "3a6e8b0e-0d3b-4c34-a0bb-83756f7b8224")
 	)
